priv/repo/seeds.dev.exs

```diff
@@ -27,6 +34,17 @@ for index <- (1..seeds_config[:squads]) do
   |> Enum.each(fn user -> insert(:squad_member, user: user, squad: squad, role: :student) end)
 end
 
+# create squad requset for users with squad
+Repo.all(User)
+|> Repo.preload(squad_member: :squad)
+|> Enum.each(fn user ->
+  if squad_member = user.squad_member do
+    build(:squad_request, user: user, squad: squad_member.squad)
+    |> approved_by(squad_member)
```

Comment:
If I'm not mistaken it means that user approved his request by himself, probably here should be a squad commander

Reply:
Yep, lost it while I was refactoring code